src/MyTested.AspNetCore.Mvc.Session/Builders/Data/WithoutSessionBuilder.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace MyTested.AspNetCore.Mvc.Builders.Data
+{
+    using Contracts.Data;
+    using Microsoft.AspNetCore.Http;
+
+    /// <summary>
+    /// Used for building <see cref="ISession"/>.
+    /// </summary>
+    public class WithoutSessionBuilder : SessionBaseBuilder, IAndWithoutSessionBuilder
```

Comment:
Here the naming is perfect. It should be consistent across the new functionality.